ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/tunnel-interfaces.j2

```diff
@@ -64,6 +70,9 @@ interface {{ tunnel_interface.name }}
 {%     if tunnel_interface.path_mtu_discovery is arista.avd.defined(true) %}
    tunnel path-mtu-discovery
 {%     endif %}
+{%     if tunnel_interface.ipsec_profile is arista.avd.defined and tunnel_interface.tunnel_mode == "ipsec" %}
```

Comment:
shouldn't we check that ip sec profile is there in ip_security.profiles. If profile is not there in device the eapi playbook fails.
```
fatal: [wan2-dc1]: FAILED! => {"changed": false, "module_stderr": "IPsec profile Profile-2 has not been created", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error"}
```